typespec-tests/src/main/java/com/cadl/patch/models/Fish.java

```diff
@@ -161,6 +158,22 @@ public int getAge() {
         return this.age;
     }
 
+    /**
+     * Set the age property: The age property.
+     * <p>
+     * Required when create the resource.
+     * </p>
+     * 
+     * @param age the age value to set.
+     * @return the Fish object itself.
+     */
+    @Generated
+    public Fish setAge(int age) {
+        this.age = age;
+        this.updatedProperties.add("age");
+        return this;
+    }
```

Comment:
Now it is via setter (instead of via ctor).

Again Javadoc mention that this is required on create.

Reply:
On the changes of the ctor -> setter, may be easier to see from the changes to test https://github.com/Azure/autorest.java/pull/2570/commits/6f108cffbf4cf24844c6efb735242e69a096ad74